### DISCUSSION TITLE
Copying a Project

### DISCUSSION THREAD

**Discussion by impact-computing** [NationalSecurityAgency/skills-service#1096] [Q&A] [answered]
Hello there skills-service team!

I was wondering if there was a way to copy a project and all the skills in it? We have the need to create a new one that is very similar to another one.

I originally thought that sharing the skills would have been the way to do it, but it seems the only thing sharing a skill does is allow it to be added in as a dependency. Maybe I am missing something.

Thanks in advance!

**Comment by sudo-may** [comment]
Hello @impact-computing 

The capability to copy a project doesn't exist yet but there is already an existing ticket for copying/syncing project's training profile ( https://github.com/NationalSecurityAgency/skills-service/issues/517). There is potential that this ticket will be tackled within the next few releases so I'd keep an eye on it. 

Also we are currently working on an ability to export/share skills to a Skills Catalog. Catalog-shared skills can then be imported as read-only into other projects. This work is done via https://github.com/NationalSecurityAgency/skills-service/issues/889 and is scoped to be available in the next release (skills-service:1.9.0).

Hope that helps!

**Comment by sudo-may** [comment, marked as answer]
Hello @impact-computing,

Exciting update to this discussion -  [skills-service:2.0](https://skilltreeplatform.dev/release-notes/skills-service.html#_2-0-0) introduced the [Copy Project Definition](https://skilltreeplatform.dev/dashboard/user-guide/projects.html#copy-project) feature - use an existing Project as a template and copy its entire training profile (subjects, skills, badges, etc..) as a new project.

Have a great day!
